### PR TITLE
externalize orientdb camel component 

### PR DESCRIPTION
**Pull request by SergeyAsm**
externalize orientdb camel component and fix "unsaved task session show in list" bug

### CODE REVIEW

File: orienteer-camel/pom.xml [hunk 1]
```diff
@@ -60,6 +65,22 @@
 			<artifactId>xom</artifactId>
 			<version>1.2.5</version>
 		</dependency>
+
+		<dependency>
+		    <groupId>org.apache.poi</groupId>
```
Comment (PhantomYdn) [inline orienteer-camel/pom.xml]: Is it really needed by default?

File: orienteer-core/src/main/java/org/orienteer/core/tasks/OTaskSessionRuntime.java [hunk 2]
```diff
@@ -33,6 +33,7 @@ public OTaskSessionRuntime() {
 	
 	public OTaskSessionRuntime(String sessionClass) {
 		persistedSession = new OTaskSessionPersisted(new ODocument(sessionClass));
+		persistedSession.save().getDocument().getDatabase().commit();
```
Comment (PhantomYdn) [inline orienteer-core/src/main/java/org/orienteer/core/tasks/OTaskSessionRuntime.java]: Why commiting of document needed here? Idea was "do not save" document unless session started.

File: orienteer-camel/pom.xml (hunk 1 above)
Comment (SergeyAsm) [inline orienteer-camel/pom.xml]: It is need for use misc documents(word,excel,etc.)

File: orienteer-core/src/main/java/org/orienteer/core/tasks/OTaskSessionRuntime.java (hunk 2 above)
Comment (SergeyAsm) [inline orienteer-core/src/main/java/org/orienteer/core/tasks/OTaskSessionRuntime.java]: You have alternative "unsaved task session show in list" bug fix?

**Pull request merged**